src/Components/Button.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import '../Styles/Button.css'
+
+function Button({text}) {
+  return (
+    <div>
+      <button>{text}</button>
+    </div>
+  )
```

Comment:
this div isn't doing anything
```suggestion
  return <button>{text}</button>;
```